### PR TITLE
Ceph: describe exposed ports

### PR DESCRIPTION
**Pull request by mtojek**
This PR adjusts Ceph docs to contain information about different exposed ports.

### CODE REVIEW

File: metricbeat/docs/modules/ceph.asciidoc [hunk 1]
```diff
@@ -6,7 +6,10 @@ This file is generated! See scripts/mage/docs_collector.go
 == Ceph module
 
 The Ceph module collects metrics by submitting HTTP GET requests to
-the http://docs.ceph.com/docs/master/man/8/ceph-rest-api/[ceph-rest-api]. The default metricsets are `cluster_disk`, `cluster_health`, `monitor_health`, `pool_disk`, `osd_tree`.
+the https://docs.ceph.com/docs/jewel/man/8/ceph-rest-api/[ceph-rest-api]. The default metricsets are `cluster_disk`, `cluster_health`, `monitor_health`, `pool_disk`, `osd_tree`.
+
+Metricsets connecting to the Ceph REST API uses by default the service exposed on port 5000.
+Metricsets using the Ceph Manager Daemon communicate with the API exposed by default on port 8003 (SSL encryption).
```
Comment (jsoriano) [inline metricbeat/docs/modules/ceph.asciidoc:12]: We could also add a block with an example config for these modules to `config.reference.yml`, maybe with some comments about what block should be added, so this appears as example configuration below. wdyt?

Comment (mtojek) [reply metricbeat/docs/modules/ceph.asciidoc:12]: Actually we have a dedicated one: `config-mgr.yml`. Do you think we should modify the reference too?

Comment (mtojek) [reply metricbeat/docs/modules/ceph.asciidoc:12]: I've updated the reference configuration.

Comment (jsoriano) [reply metricbeat/docs/modules/ceph.asciidoc:12]: :+1: thanks

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -310,6 +310,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add test for documented fields check for metricsets without a http input. {issue}17315[17315] {pull}17334[17334]
 - Add final tests and move label to GA for the azure module in metricbeat. {pull}17319[17319]
 - Reference kubernetes manifests mount data directory from the host when running metricbeat as daemonset, so data persist between executions in the same node. {pull}17429[17429]
+- Improve reference configuration for Ceph module. {pull}17566[17566]
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc:313]: No need to add a changelog for this.

Comment (mtojek) [reply CHANGELOG.next.asciidoc]: Ok, removed.

**Pull request merged**